package.json

```diff
@@ -22,7 +22,9 @@
     "test": "ava '*.test.js' '**/*.test.js' '!integration_test/*'",
     "test:integration": "npm run db:init && ava --concurrency=1 integration_test/*.test.js",
     "test:cover": "nyc --reporter=text-lcov ava --concurrency=1 '*.test.js' '**/*.test.js' > coverage.lcov && codecov",
+    "test:inta": "npm run db:init && ava --concurrency=1 integration_test/auth.integration.test.js",
     "start": "node ./index.js",
+    "debug": "node --nolazy --inspect-brk=9229 ./index.js",
```

Comment:
@wojta Was this accidentally checked in?

Reply:
yes